### ISSUE TITLE
Modify/delete playlists

### ISSUE THREAD

**Issue by ericshu6** [ppy/osu#9565]
**Describe the new feature:** 
The ability to delete/modify timeshift rooms.
**Proposal designs of the feature:** 
You can right-click the room to edit/delete it

**Comment by peppy** [comment]
Not a huge deal, but we do need to figure out how long to allow edits for, and what to do with existing scores.

**Comment by gagahpangeran** [comment]
Maybe as long there's no score submitted. The example case is you already create a room and want to change the playlist quickly.

**Comment by kathelynn** [comment]
Could also be useful if the room can be ended before the duration, and a results screen to show old matches to see how they went~
(also limit to something like 20 results and then require osu!supporter for more lol, ideas for you peppy)

**Comment by r00ster91** [comment]
I think this would reduce the amount of spammy/accidental playlists a lot and increase quality because I also just made the mistake of posting my playlist before I wanted to and now I can't delete it.

It should be possible to delete for as long as there are no scores.

**Comment by peppy** [comment]
Yep, deleting when there's no scores should be a good starting point.

**Comment by Joehuu** [comment]
Probably resolved with close grace period in https://github.com/ppy/osu/pull/30793, closing.
